www/js/lib/uiUtil.js

```diff
@@ -181,6 +205,7 @@ define([], function() {
     return {
         feedNodeWithBlob: feedNodeWithBlob,
         replaceCSSLinkWithInlineCSS: replaceCSSLinkWithInlineCSS,
+        URL: deriveURL,
```

Comment:
It would probably be better to expose the same function name to the outside as the original function name

Reply:
My idea in exposing it as "URL" was that I'm doing a poorman's version of a URL polyfill for IE11. If we weren't supporting IE11, we wouldn't actually need a function to do this, we could easily just use the URL API.

Given this reasoning, would you still prefer me to expose it publicly with the new name for the function (`uiUtil.deriveZimUrlFromRelativeUrl`), or keep `uiUtil.URL` in order to show its relationship to the URL API? (The reason I can't call it `URL` inside `uiUtil.js` is because it would conflict with the URL API.)

I did look into adding this function into the prototype of the URL function for IE11, but it looked like some extra coding for the same result, and it didn't seem justified.

Reply:
I understand, but it's not really a polyfill : as you said, it also has some ZIM-specific logic.
So I'd stick to a classical function.
Another option would be to use a real polyfill, but I agree with you that it's probably overkill for this simple need

Reply:
OK, done!